machine/machine_core/ssh_connection.py

```diff
@@ -145,7 +145,7 @@ def wait_reboot(self, timeout_sec=180):
                 if self.boot_id != boot_id:
                     break
             except exceptions.Failure:
-                pass
+                pass  # try again
```

Comment:
any comment makes codeQL happy? :)

Reply:
That's at least what the [detailed view claims](https://github.com/cockpit-project/bots/security/code-scanning/29). Once this lands, I'll rebase #4039, then we'll see whether it really worked. :grin: